### PR TITLE
Form explainer new content and boxes

### PR DESCRIPTION
**Pull request by stephanieosan**
### Additions
* Added content to checklist category for pages 2 and 3 of Loan Estimate Explainer
* Added content to checklist category for page 4 of Closing Disclosure Explainer

### Screenshots
screen shot 2015-08-25 at 11 11 09 am
screen shot 2015-08-25 at 11 11 14 am
screen shot 2015-08-25 at 11 10 54 am

### Review
@virginiacc or @cfarm or @amymok 
and @mthibos could you check that all the boxes are there and in the right spots in those screenshots? Then you can check the full content on build.

### CODE REVIEW

**Comment by mthibos** [comment]
@sosan looks great!  The one box I would change is the escrow box on the closing disclosure, since the checkbox for if you don't have an escrow is at the bottom of that section.

image

**Comment by stephanieosan** [comment]
That makes sense and is now updated!

screen shot 2015-08-25 at 4 22 36 pm

**Comment by mthibos** [comment]
:+1:

File: src/loan-estimate/loan-estimate-2.html [hunk 1]
```diff
@@ -3,6 +3,86 @@
 "img": url_for('static', filename='img/loan-estimate-H24B-2.png'),
 'terms':
 [
+        {
+        "term": "Compare the Originiation Charges to Loan Estimates from other lenders",
```
Comment (virginiacc) [inline src/loan-estimate/loan-estimate-2.html]: Looks like there's a typo in 'Origination'.

**Comment by virginiacc** [comment]
This looks great! I found one typo, but I think other than that it's ready to merge. :shipit:

**Comment by stephanieosan** [comment]
Awesome, thanks for catching that! It should be fixed now.

**Comment by virginiacc** [comment]
:+1:

**Pull request merged**